### PR TITLE
Update to latest contracts and add one tx method

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

This pull request updates colonyJS methods and events to colonyNetwork commit [9bba127](https://github.com/JoinColony/colonyNetwork/commit/9bba127b0286708d4f8919526a943b0e916cfd7c) and adds the temporary one transaction payment method in the [temp/eth-denver](https://github.com/JoinColony/colonyNetwork/commits/temp/eth-denver) branch (commit [8888fd4](https://github.com/JoinColony/colonyNetwork/commit/8888fd47814ca9d24eb750c2f1b7ed24576a1744)).

## Checklist

- [x] Update colonyJS methods and events to colonyNetwork commit `9bba127`
- [x] Add the one transaction payment method in the `temp/eth-denver` branch

### CODE REVIEW

File: packages/colony-js-client/src/ColonyClient/senders/MakePayment.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export default class MakePayment extends ContractClient.Sender<
+  InputValues,
+  OutputValues,
+  ColonyClient,
+  *,
+> {
+  async _sendTransaction(args: *, options: *) {
+    const colony = this.client.contract.address;
+    const contract = await this._getContract(args);
```
Comment (JamesLefrere) [inline packages/colony-js-client/src/ColonyClient/senders/MakePayment.js]: I think this should be called without an argument.

File: packages/colony-js-client/src/ColonyClient/senders/MakePayment.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    args.unshift(colony);
```
Comment (JamesLefrere) [inline packages/colony-js-client/src/ColonyClient/senders/MakePayment.js]: It's usually not a good idea to mutate function arguments, so I would suggest this change:

```js
    const contract = await this._getContract();
    return contract.callTransaction(
      'makePayment',
      // the first argument must be the colony address
      [this.client.contract.address, ...args],
      options,
    );
```

File: packages/colony-js-client/src/ColonyClient/senders/MakePayment.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    return contract.callTransaction('makePayment', args, options);
+  }
+
+  async _getContract(args: *) {
+    return this.client.adapter.getContract(
+      { contractName: 'OneTxPayment' },
+      args,
```
Comment (JamesLefrere) [inline packages/colony-js-client/src/ColonyClient/senders/MakePayment.js]: Flow tells me this only accepts one argument (the query object), so I think what we need here is just the `contractName` – AFAIK there's only ever one `OneTxPayment` contract.

File: packages/colony-js-client/src/ColonyClient/senders/MakePayment.js (hunk 1 above)
Comment (ryanchristo) [reply packages/colony-js-client/src/ColonyClient/senders/MakePayment.js]: 👍

File: packages/colony-js-client/src/ColonyClient/senders/MakePayment.js (hunk 2 above)
Comment (ryanchristo) [reply packages/colony-js-client/src/ColonyClient/senders/MakePayment.js]: 👍

File: packages/colony-js-client/src/ColonyClient/senders/MakePayment.js (hunk 3 above)
Comment (ryanchristo) [reply packages/colony-js-client/src/ColonyClient/senders/MakePayment.js]: 👍

**Pull request merged**